app/src/main/java/org/simple/clinic/patient/recent/RecentPatientsView.kt

```diff
@@ -5,12 +5,30 @@ import android.util.AttributeSet
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
 import com.xwray.groupie.GroupAdapter
+import io.reactivex.Observable
+import io.reactivex.rxkotlin.subscribeBy
+import org.simple.clinic.activity.TheActivity
 import org.simple.clinic.patient.Gender
+import org.simple.clinic.patient.PatientConfig
+import javax.inject.Inject
 
 class RecentPatientsView(context: Context, attrs: AttributeSet) : RecyclerView(context, attrs) {
 
+  @Inject
+  lateinit var config: Observable<PatientConfig>
+
   override fun onFinishInflate() {
     super.onFinishInflate()
+    TheActivity.component.inject(this)
+
+    config.subscribeBy(onNext = {
```

Comment:
Move the config to either `HomeScreen` or `HomePagerAdapter`

Reply:
Added the config check to `PatientsScreenKey`.